perfrunbook/utilities/capture_flamegraphs.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+help_msg() {
+ echo "usage: $0 oncpu|offcpu [time seconds|default 300]"
+}
+
+capture_on_cpu () {
+  perf record -a -g -k 1 -F99 -e cpu-clock:pppH -- sleep $1
```

Comment:
you should check that perf exists in the path and is runnable and print help if not